modules/videoio/include/opencv2/videoio/videoio_c.h

```diff
@@ -391,6 +393,22 @@ enum
     CV_CAP_INTELPERC_IMAGE                  = 3
 };
 
+// gPhoto2 properties, if propertyId is less than 0 then work on widget with that __additive inversed__ camera setting ID
+// Get IDs by using CAP_PROP_GPHOTO2_WIDGET_ENUMERATE.
+// @see CvCaptureCAM_GPHOTO2 for more info
+enum {
```

Comment:
Same here. However I am not sure that you should make the fixes to this _c header. It is mainly there for backwards compatibility but as far as I know we do not add new functionality in the C interface. @vpisarev could you clarify this?